### DISCUSSION TITLE
Add other constructed languages

### DISCUSSION THREAD

**Discussion by 002gRTEah3mRYH** [stoatchat/discussions#146] [Feature Suggestions]
Hello,

I saw that we can select Toki Pona, but why not esperanto ? 
And other languages from The Lord of The Ring, Game of Thrones, Star Trek ?

**Comment by insertish** [comment]
Unless if you're willing to translate those languages, nobody is available to do so.
Toki Pona was added on request and there was a few people who helped translate it.
